ui/src/store/device-control-store.ts

```diff
@@ -38,6 +38,102 @@ export class DeviceControlStore {
     this.deviceControlService.back()
   }
 
+  pressPower(): void {
```

Comment:
I'm not sure we need to have so many methods with a single line that calls another function. Maybe we can use `new Proxy` to override attribute lookup and if it's a known attribute of deviceControlService - call it by `[]`. 

I'm worried adding new methods or buttons might be cumbersome